cassdegrees/templates/widgets/rulescripts.html

```diff
@@ -76,10 +76,15 @@
             <label>
                 Students must select...
             </label>
-            <input class="text" onkeydown="javascript: return checkKeys(event)" type="number" v-on:change="check_options" v-model="details.unit_count" min="0" step="6" max="1000" aria-required="true" required>
-            units
         </p>
 
+         <select v-model="details.list_type" required>
```

Comment:
There doesn't appear to be error checking for this one - it can be saved with an empty value.

Reply:
fixed with latest batch of commits